lxd/network/openvswitch/ovn.go

```diff
@@ -1269,6 +1270,10 @@ func (o *OVN) LogicalSwitchPortAdd(switchName OVNSwitch, portName OVNSwitchPort,
 		if opts.Location != "" {
 			args = append(args, "--", "set", "logical_switch_port", string(portName), fmt.Sprintf("external_ids:%s=%s", ovnExtIDLXDLocation, opts.Location))
 		}
+
+		if opts.VLAN != 0 {
```

Comment:
If opts.VLAN == 0 we should unset the tag_request field (and confirm that doing that clears the tag field set by OVN) so that one can disable the VLAN setting later.